<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xstrato_ui</class>
 <widget class="QMainWindow" name="xstrato_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image_display">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>701</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/assets/space.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="test_button">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>540</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 2px solid;
border-color: rgb(215, 237, 252);
border-radius: 10px;
}
QPushButton:hover {
background-color: black; 
border: 3px solid;
border-color: rgb(215, 237, 252);
}
QPushButton:pressed {
border: 4px solid;
border-color: rgb(92, 242, 135);
}
</string>
    </property>
    <property name="text">
     <string>Test button</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>410</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

image: url(:/assets/krtek.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Received image </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial port status</string>
    </property>
   </widget>
   <widget class="QLabel" name="serialport_status">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/assets/refresh.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="open_serial">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Serial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_serial">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Serial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_color_cmd">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
qproperty-icon: url(:/assets/send.png);
qproperty-iconSize: 50px;
border-radius: 25px;
}
QPushButton:hover {
background-color: rgb(10, 0, 0); 
}
QPushButton:pressed {
background-color: rgb(16, 16, 16);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="color_edit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>200</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nbr_rx_packet">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="nbr_tot_packet">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="nbr_tot_packet_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>240</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="file_transmission_progress_bar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	color: white;
	font: 16pt &quot;Times New Roman&quot;;
	height: 30px;
}
QProgressBar:horizontal {
	border: 1px solid gray;
	border-radius: 10px;
	background: black;
	padding: 1px;
    text-align: center;
}
QProgressBar::chunk:horizontal {
	background: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 		0.5, stop: 0 rgb(39, 68, 255), stop: 1 rgb(255, 38, 114));
	border-radius: 10px;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%v / %m</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send color CMD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_image">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>340</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;my button&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
qproperty-icon: url(:/assets/clear.png);
qproperty-iconSize: 30px;
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-radius: 25px;
}
QPushButton:hover {
background-color: black; 
}
QPushButton:pressed {
border: 1px solid;
border-color: rgb(101, 156, 173);
}
</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_since_last_Rx">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Last packet's timestamp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>2:32</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="last_refresh_label">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>10</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Last packet's timestamp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Time since last received packet</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="packet_rate_bar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	color: white;
	font: 10pt;
}
QProgressBar:horizontal {
	border: 1px solid gray;
	border-radius: 10px;
	background: black;
	padding: 2px;
    text-align: center;
	height: 10px;
}
QProgressBar::chunk:horizontal {
	background: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 		0.5, stop: 0 rgb(39, 68, 255), stop: 1 rgb(255, 158, 110));
	border-radius: 5px;
}</string>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%v packets/sec</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="tank_N2O">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>300</y>
      <width>101</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	color: white;
	font: 16pt &quot;Times New Roman&quot;;
	height: 30px;
}
QProgressBar {
	border: 3px solid gray;
	border-radius: 50px;
	background: black;
	padding: 2px;
    text-align: center;
}
QProgressBar::chunk {
	background: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5, stop: 0 rgb(39, 68, 255), stop: 1 rgb(255, 38, 114));
    border-bottom-right-radius: 44px;
    border-bottom-left-radius: 44px;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(43, 76, 199, 255), stop:1 rgba(193, 0, 255, 255));
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgba(240, 248, 255, 255),
                                   stop: 0.4 rgba(19, 170, 249, 255),
                                   stop: 0.7 rgba(8, 104, 208, 255),
                                   stop: 1 rgba(28, 32, 127, 255));

    /*height: 10px;
    margin: 1px;*/
}</string>
    </property>
    <property name="value">
     <number>76</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="format">
     <string>%v%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>510</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: white;
	font: 16pt &quot;Calibri Light&quot;;</string>
    </property>
    <property name="text">
     <string>N2O Tank</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial port detected:</string>
    </property>
   </widget>
   <widget class="QLabel" name="serial_port_detected_name">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
